gulpfile.js

```diff
@@ -57,7 +57,9 @@ gulp.task('min-qt', ['qt'], function() {
 
 gulp.task('build', ['qt', 'min-qt']);
 
-gulp.task('test', ['build'], function(done) {
+gulp.task('build-dev', ['qt']);
+
+gulp.task('test', ['build-dev'], function(done) {
```

Comment:
Hm. I undertand why you want this, but this shouldn't go under the name `test`.
`test` should be full tests, and those should probably include testing the minification process.

For example in #57 `min-qt` failed on two commits due to mistypes. Those commits would have failed later, but I am not sure that it would be true for all cases.

Note that `qt-min` stage is not what is slowing down the CI.

Perhaps this should be a separate stage? Like `test-fast`?